pkg/provider/aws/resources/cloudfront.go

```diff
@@ -176,6 +147,41 @@ func NewCloudfrontDistribution(appName string, cdnId string) *CloudfrontDistribu
 	}
 }
 
+type CloudfrontDistributionCreateParams struct {
+	CdnId   string
+	AppName string
+	Refs    core.AnnotationKeySet
+}
+
+func (distro *CloudfrontDistribution) Create(dag *core.ResourceGraph, params CloudfrontDistributionCreateParams) error {
+	distro.Name = cloudfrontDistributionSanitizer.Apply(fmt.Sprintf("%s-%s", params.AppName, params.CdnId))
+	distro.ConstructsRef = params.Refs
+
+	if dag.GetResource(distro.Id()) != nil {
+		return errors.Errorf(`duplicate Cloudfront distribution "%s" (internal error)`, distro.Id())
+	}
+	dag.AddResource(distro)
+
+	// Some defaults. Someday these may move to a Configure (and be configurable), but not yet.
+	distro.DefaultCacheBehavior = &DefaultCacheBehavior{
+		AllowedMethods: []string{"DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"},
+		CachedMethods:  []string{"HEAD", "GET"},
+		ForwardedValues: ForwardedValues{
+			QueryString: true,
+			Cookies:     Cookies{Forward: "none"},
+		},
+		MinTtl:               0,
+		DefaultTtl:           3600,
+		MaxTtl:               86400,
+		ViewerProtocolPolicy: "allow-all",
+	}
+	distro.Restrictions = &Restrictions{
+		GeoRestriction: GeoRestriction{RestrictionType: "none"},
+	}
```

Comment:
i would put all these configuration like things in the Configure function just so we can expose it in the future

Reply:
It ended up being non-trivial to do that, so I filed a bug and am going to keep things as-is: https://github.com/klothoplatform/klotho/issues/652

Reply:
im a little confused as to why its non-trivial. In the edge couldnt you just set the specific thing on the DefaultCacheBehavior? Then configure will set the remainder of the fields?

Reply:
this will be the same in expose also, the default cache would be nil. I would just add a method that sets the defaultTargetId for the cdn, if the default cache is nil create and set the field. If its not nil, just override the field.

Then in configure we would take have to just not set that field in the default cache behavior because we know its derived by edge expansion